js/src/utils/localStorage.js

```diff
@@ -0,0 +1,22 @@
+const localStorageExists =
+	typeof window !== 'undefined' && 'localStorage' in window;
+
+const localStorage = {
+	get( key ) {
+		return localStorageExists ? window.localStorage.getItem( key ) : null;
```

Comment:
💅  What do you think about...

`return window?.localStorage?.getItem( key )`  or `return window?.localStorage?.getItem( key ) ?? null` (in case we need null as return for a reason). 

That way we can remove both conditionals and the localStorageExists computed function